<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Systemix Capital - Trend Recall</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_trend.jpg</normaloff>icon_trend.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Pasado (Debe estár seleccionado el archivo historicos)</string>
    </property>
    <widget class="QDateEdit" name="date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>2</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2013</year>
       <month>2</month>
       <day>5</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>2</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2020</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1952</year>
       <month>9</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2013</year>
       <month>2</month>
       <day>5</day>
      </date>
     </property>
    </widget>
    <widget class="QTimeEdit" name="hora">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="time">
      <time>
       <hour>10</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dia de prueba</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hora de tope</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_graficar_test">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Graficar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Principal</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Porcentajes</string>
     </property>
     <widget class="QSpinBox" name="p_gap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="p_simil">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>% del gap</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>% de 
similitud</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="b_graficar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Graficar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_historicos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="x">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>14</number>
     </property>
    </widget>
    <widget class="QPushButton" name="b_dactual">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dia actual</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>76</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Indicador a </string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_historicos">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Historicos</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="y">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>14</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="dias_graf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QComboBox" name="Indicador">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>RSI(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STO(x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STO(EMA(y),x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RSI(EMA(y), x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STO(DF(),x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RSI(DF(),x)</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="line_dactual">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>a graficar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cantidad de dias </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>89</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>comparar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
